tests/api/fixtures/tasks.ts

```diff
@@ -1,6 +1,5 @@
-import { map } from 'lodash-es'
 import { createTasksInBulk } from '#controllers/tasks/lib/tasks'
-import { checkEntities } from '../utils/tasks.js'
```

Comment:
`createSomeTasks` still seems to use those dependencies, but since 43aa8b7 there are no consumer for `createSomeTasks` so it can also be removed I guess

Reply:
done in 7d0a14873